docs/frontend/webpacker.md

```diff
@@ -37,6 +37,15 @@ loading split chunks multiple times.
 <%= javascript_packs_with_chunks_tag "webShare", "someOtherPack", defer: true %>
 ```
 
+## Webpack aliases
+
+The project uses
+[webpack aliases](https://webpack.js.org/configuration/resolve/#resolvealias).
+The aliases used in the project can be found in
+https://github.com/forem/forem/blob/master/config/webpack/environment.js#L21-L26
```

Comment:
```suggestion
The aliases used in the project can be found under `alias` in
https://github.com/forem/forem/blob/master/config/webpack/environment.js
```

(suggestion / not blocking) Just an idea as the line reference is already going to be out of date once this PR is merged